apollo-mockserver/src/commonMain/kotlin/com/apollographql/apollo3/mockserver/MockServer.kt

```diff
@@ -26,7 +33,12 @@ interface MockServerInterface {
   fun takeRequest(): MockRecordedRequest
 }
 
+abstract class BaseMockServer(override val mockDispatcher: MockDispatcher) : MockServerInterface {
+  override fun enqueue(mockResponse: MockResponse) {
+    (mockDispatcher as QueueMockDispatcher).enqueue(mockResponse)
```

Comment:
```suggestion
    (mockDispatcher as? QueueMockDispatcher)?.enqueue(mockResponse) ?: error("Apollo: cannot call MockeServer.enqueue() with a custom dispatcher")
```

(or s/dispatcher/handler if we decide to rename)

Reply:
In retrospect, I'm not a huge fan of the `MockServer` naming either because of the possible "mockito" confusion but certainly too late to change this 🤷